### PR TITLE
add test for UCR reconciliation mapping

### PR DESCRIPTION
**Pull request by snopoke**
##### SUMMARY
Building on what @sravfeyn did here: https://github.com/dimagi/commcare-hq/pull/27157

Ensure that the recon task is checking all the right forms / cases.

### CODE REVIEW

File: custom/icds_reports/models/util.py [hunk 1]
```diff
@@ -93,6 +93,72 @@ def run_aggregation_queries(self) -> bool:
...
+            'http://openrosa.org/formdesigner/1D568275-1D19-46DB-8C54-2C9765DF6335',
+            'http://openrosa.org/formdesigner/362f76b242d0cfdcec66776f9586dc3620e9cce5',
+            'http://openrosa.org/formdesigner/756ec44475658f3f463f8012632def2bc9fbe731',
+            'http://openrosa.org/formdesigner/66d52f84d606567ea29d5fae88f569d2763b8b62',
+            'http://openrosa.org/formdesigner/b183124a25f2a0ceab266e4564d3526199ac4d75',
+        ],
+        "static-dashboard_growth_monitoring_forms": [
+            'http://openrosa.org/formdesigner/7a55754119359466b1951d7251068bd4f45e73c3',
+            'http://openrosa.org/formdesigner/b183124a25f2a0ceab266e4564d3526199ac4d75',
+            'http://openrosa.org/formdesigner/376FA2E1-6FD1-4C9E-ACB4-E046038CD5E2',
```
Comment (snopoke) [inline custom/icds_reports/models/util.py:115]: These 2 were added in this PR

**Review by calellowitz — APPROVED** [review]
This PR is an interesting change because the original work was predicated on the notion that missing forms represented forms that did not make it into kafka. As such, we designed this to only check each XMLNS in one data source, under the assumption that if it was in one, it made it into all. The recent issues indicate that isnt true. The missing form in the JIRA ticket is in the usage data source, which is why it was not picked up by this task. This adds a bunch more work to the task, since we get a lot of gm forms each day, but it doesn't really seem avoidable.

**Comment by snopoke** [comment]
@calellowitz based on my understanding of the pillow processing I think it's perfectly reasonable that a doc may get processed by one UCR and error in another. The errors should get retried or end up in the pillow error queue though so not sure where that's failing.

File: custom/icds_reports/models/util.py [hunk 2]
```diff
@@ -93,6 +93,72 @@ def run_aggregation_queries(self) -> bool:
         return False
 
 
+UCR_MAPPING = {
```
Comment (sravfeyn) [inline custom/icds_reports/models/util.py:96]: Is there any reason to not to generate this mapping from `StaticDataSourceConfiguration` dynamically?

Comment (snopoke) [reply custom/icds_reports/models/util.py:96]: Yea, at this point I guess it could be.

File: custom/icds_reports/models/util.py (hunk 1 above)
Comment (sravfeyn) [reply custom/icds_reports/models/util.py:115]: The form that Cal mentioned in the ticket falls under this previously missing xmlns https://dimagi-dev.atlassian.net/browse/IIO-827

**Pull request merged**